### ISSUE TITLE
指令精调问题

### ISSUE THREAD

**Issue by sunshineyg2018** [ymcui/Chinese-LLaMA-Alpaca#465]
使用zero3

torchrun --nnodes 1 --nproc_per_node 3 run_clm_sft_with_peft.py \
    --deepspeed ds_zero2_no_offload.json \
    --model_name_or_path \
    --tokenizer_name_or_path \
    --dataset_dir  \
    --validation_split_percentage 0.001 \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --do_train \
    --do_eval \
    --seed $RANDOM \
    --fp16 \
    --max_steps 5200 \
    --lr_scheduler_type cosine \
    --learning_rate 1e-4 \
    --warmup_ratio 0.03 \
    --weight_decay 0 \
    --logging_strategy steps \
    --logging_steps 10 \
    --save_strategy steps \
    --save_total_limit 3 \
    --evaluation_strategy steps \
    --eval_steps 250 \
    --save_steps 500 \
    --gradient_accumulation_steps 1 \
    --preprocessing_num_workers 8 \
    --max_seq_length 512 \
    --output_dir /home/ubuntu/train_code/m \
    --overwrite_output_dir \
    --ddp_timeout 30000 \
    --logging_first_step True \
    --torch_dtype float16 \
    --validation_file  \
    --peft_path \
    --gradient_checkpointing \
    --ddp_find_unused_parameters False

发送错误 
: DeepSpeed Zero-3 is not compatible with `low_cpu_mem_usage=True` or with passing a `device_map`.ValueError
: DeepSpeed Zero-3 is not compatible with `low_cpu_mem_usage=True` or with passing a `device_map`.

**Comment by AliceShen122** [comment]
我也遇到了这个问题，请问什么原因呢？

**Comment by lyccyl1** [comment]
set ` low_cpu_mem_usage=False` or 注释掉`device_map`
